### ISSUE TITLE
Can't get photo's name just taken with camera

### ISSUE THREAD

**Issue by leapit** [Sh1d0w/multi_image_picker#116]
**Bug Report**

`multi_image_picker` version: 4.0.1

Is it happening on Android: Yes

Is it happening on iOS: Yes

```
[✓] Flutter (Channel stable, v1.2.1, on Mac OS X 10.13.6 17G6030, locale en-CN)
[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
[✓] iOS toolchain - develop for iOS devices (Xcode 10.1)
[✓] Android Studio (version 3.3)
[✓] VS Code (version 1.32.3)
[✓] Connected device (1 available) 

```

**Describe the bug**

Use basic usage [https://sh1d0w.github.io/multi_image_picker/#/basicusage](https://sh1d0w.github.io/multi_image_picker/#/basicusage),and take photo first,then select this photo just token,and print:

```
    for (var val in resultList) {
      print(val.name);  ----> empty
      generateThumb(val);
    }
```
no photo name.

It happened only this photo which just token,to common photo select will console out the photo name.

**Comment by Sh1d0w** [comment]
Thanks for the report, this is now fixed in version 4.0.3

**Comment by leapit** [comment]
@Sh1d0w Thanks,tried,and worked!
